api/envoy/service/ratelimit/v3/rls.proto

```diff
@@ -49,6 +49,8 @@ message RateLimitRequest {
 
   // Rate limit requests can optionally specify the number of hits a request adds to the matched
   // limit. If the value is not set in the message, a request increases the matched limit by 1.
+  // This value can be overridden by setting filter state value `envoy.ratelimit.hits_addend`
```

Comment:
```
``envoy.ratelimit.hits_addend``
```

Please fix the CI, these is a issue that I know. Double backticks should be used.